jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java

```diff
@@ -233,11 +236,11 @@ private HandlerCollection getHandlerCollection(
         GzipHandler gzipHandler = newGzipHandler(serverConfig);
         gzipHandler.setHandler(servletContextHandler);
 
-        StatisticsHandler statisticsHandler = newStatisticsHandler();
-        statisticsHandler.setHandler(gzipHandler);
+        HttpResponseStatisticsCollector statisticsCollector = new HttpResponseStatisticsCollector();
+        statisticsCollector.setHandler(gzipHandler);
 
         HandlerCollection handlerCollection = new HandlerCollection();
-        handlerCollection.setHandlers(new Handler[]{statisticsHandler});
+        handlerCollection.setHandlers(new Handler[] { statisticsCollector });
```

Comment:
Replacing the `StatisticsHandler` may affect the statistics Jetty/JDisc exports through JMX (when configured). Is it possible to keep the `StatisticsHandler`?

Reply:
I returned `StatisticsHandler` to the processing chain in the latest commit, since as you mentioned it is accessible through other mechanisms.